<logs><log LogGUID="992f8d87-c3a7-46fb-bdb5-84be9eb47fc5" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>bfb6f289-407f-459a-93ce-b117341b65ad</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>RAMuU+ZdRJ1w64VxHM7fhg==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Host.ServerController.GetServersCallBack(CacheItemArgs cacheItemArgs)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromRuntimeCache(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="72c1d025-52b4-41d3-8676-b60236a155c4" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>ec2e908a-13c5-43bb-8371-690ae14af268</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>RAMuU+ZdRJ1w64VxHM7fhg==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Host.ServerController.GetServersCallBack(CacheItemArgs cacheItemArgs)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromRuntimeCache(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="de6fd4ab-a6ab-4823-a0d1-a9760c05907e" LogFileID="" LogTypeKey="SCHEDULER_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>a8ad8f5c-2150-40ab-b7af-4f617675a0bb</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>fmC+U2a98GtfvsPgW88WiQ==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at PetaPoco.Database.ExecuteScalar[T](String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteScalar[T](String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Data.DataProvider.AddScheduleHistory(Int32 ScheduleID, DateTime StartDate, String Server)
   at DotNetNuke.Services.Scheduling.Scheduler.CoreScheduler.AddScheduleHistory(ScheduleHistoryItem scheduleHistoryItem)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="84b31b30-6564-450b-839f-dc45889f0041" LogFileID="" LogTypeKey="SCHEDULER_EVENT_STARTED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>THREAD ID</PropertyName><PropertyValue>29</PropertyValue></LogProperty><LogProperty><PropertyName>TYPE</PropertyName><PropertyValue>DotNetNuke.Services.Social.Messaging.Scheduler.CoreMessagingScheduler, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>SOURCE</PropertyName><PropertyValue>STARTED_FROM_TIMER</PropertyValue></LogProperty><LogProperty><PropertyName>ACTIVE THREADS</PropertyName><PropertyValue>3</PropertyValue></LogProperty><LogProperty><PropertyName>FREE THREADS</PropertyName><PropertyValue>-2</PropertyValue></LogProperty><LogProperty><PropertyName>READER TIMEOUTS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>WRITER TIMEOUTS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>IN PROGRESS</PropertyName><PropertyValue>3</PropertyValue></LogProperty><LogProperty><PropertyName>IN QUEUE</PropertyName><PropertyValue>1</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="2399a205-ad4d-4cc2-bc65-941b13b22836" LogFileID="" LogTypeKey="SCHEDULER_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>e4ec54dc-29ef-47c9-8c54-dc53235fea1a</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>6ztB1wiNuSAKhh5C3rlwbQ==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at PetaPoco.Database.Execute(String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteNonQuery(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Data.DataProvider.UpdateScheduleHistory(Int32 ScheduleHistoryID, DateTime EndDate, Boolean Succeeded, String LogNotes, DateTime NextStart)
   at DotNetNuke.Services.Scheduling.SchedulingController.UpdateScheduleHistory(ScheduleHistoryItem objScheduleHistoryItem)
   at DotNetNuke.Services.Scheduling.Scheduler.CoreScheduler.UpdateScheduleHistory(ScheduleHistoryItem scheduleHistoryItem)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="91075edf-1160-4511-84c9-24495ea568bd" LogFileID="" LogTypeKey="SCHEDULER_EVENT_COMPLETED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>TYPE</PropertyName><PropertyValue>DotNetNuke.Services.Social.Messaging.Scheduler.CoreMessagingScheduler</PropertyValue></LogProperty><LogProperty><PropertyName>THREAD ID</PropertyName><PropertyValue>29</PropertyValue></LogProperty><LogProperty><PropertyName>NEXT START</PropertyName><PropertyValue>8/13/2015 2:36:41 AM</PropertyValue></LogProperty><LogProperty><PropertyName>SOURCE</PropertyName><PropertyValue>STARTED_FROM_TIMER</PropertyValue></LogProperty><LogProperty><PropertyName>ACTIVE THREADS</PropertyName><PropertyValue>2</PropertyValue></LogProperty><LogProperty><PropertyName>FREE THREADS</PropertyName><PropertyValue>-1</PropertyValue></LogProperty><LogProperty><PropertyName>READER TIMEOUTS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>WRITER TIMEOUTS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>IN PROGRESS</PropertyName><PropertyValue>2</PropertyValue></LogProperty><LogProperty><PropertyName>IN QUEUE</PropertyName><PropertyValue>2</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="bc7b42fb-39d4-4cb9-9f69-3eac586c8bfd" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>5aa6cb46-ed45-47da-a26b-eace54390bb4</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>zGXns7D6oOO7TCYELusg7g==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Host.ServerController.GetServersCallBack(CacheItemArgs cacheItemArgs)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromRuntimeCache(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="62602152-ea90-4f95-a02e-961fc3d7bdcd" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>c6650541-047f-4e41-8b62-145fea276f1f</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>zGXns7D6oOO7TCYELusg7g==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Host.ServerController.GetServersCallBack(CacheItemArgs cacheItemArgs)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromRuntimeCache(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="4ce666a4-7a91-4903-b406-96cb0255ccf3" LogFileID="" LogTypeKey="SCHEDULER_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>e35789c1-0df7-4f2c-adbe-e2a506b4b49b</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>fmC+U2a98GtfvsPgW88WiQ==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at PetaPoco.Database.ExecuteScalar[T](String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteScalar[T](String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Data.DataProvider.AddScheduleHistory(Int32 ScheduleID, DateTime StartDate, String Server)
   at DotNetNuke.Services.Scheduling.Scheduler.CoreScheduler.AddScheduleHistory(ScheduleHistoryItem scheduleHistoryItem)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="db7547f9-5071-4ea3-bae3-1f0ea91afbe7" LogFileID="" LogTypeKey="SCHEDULER_EVENT_STARTED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>THREAD ID</PropertyName><PropertyValue>16</PropertyValue></LogProperty><LogProperty><PropertyName>TYPE</PropertyName><PropertyValue>DotNetNuke.Services.ModuleCache.PurgeModuleCache, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>SOURCE</PropertyName><PropertyValue>STARTED_FROM_TIMER</PropertyValue></LogProperty><LogProperty><PropertyName>ACTIVE THREADS</PropertyName><PropertyValue>2</PropertyValue></LogProperty><LogProperty><PropertyName>FREE THREADS</PropertyName><PropertyValue>-1</PropertyValue></LogProperty><LogProperty><PropertyName>READER TIMEOUTS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>WRITER TIMEOUTS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>IN PROGRESS</PropertyName><PropertyValue>2</PropertyValue></LogProperty><LogProperty><PropertyName>IN QUEUE</PropertyName><PropertyValue>2</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="430365b4-d548-4c5b-94a2-adc6b56c19e3" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>3a0bc1e6-e4b1-4cd3-a0b3-e490f0a02fe3</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>RtXK48OmBzd0qFr30sbuhA==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Data.DataProvider.GetPortals(String CultureCode)
   at DotNetNuke.Entities.Portals.PortalController.GetPortalsCallBack(CacheItemArgs cacheItemArgs)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromRuntimeCache(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="0a15dece-53a7-4fc5-b475-410ac81a1b0d" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>18c36ebb-c17b-48f9-b8bc-b944dc303388</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>ke/Xsn9T7QlzXsGVjeXAMQ==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Host.ServerController.GetServersCallBack(CacheItemArgs cacheItemArgs)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromRuntimeCache(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="f433d38f-9c2e-411a-87b3-d7a1e4204443" LogFileID="" LogTypeKey="CACHE_REMOVED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:33 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>DNN_GetLogTypeConfigInfoByKey</PropertyName><PropertyValue>Removed</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="25d24fdb-d000-4194-b36f-5ec57d3153cd" LogFileID="" LogTypeKey="CACHE_REMOVED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>DNN_Tab_Tabs-1</PropertyName><PropertyValue>Removed</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="6c5a9f3f-2e93-474c-a858-3ace7ec34696" LogFileID="" LogTypeKey="CACHE_REMOVED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>DNN_PortalGroups</PropertyName><PropertyValue>Removed</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="b829781b-b226-42fa-a53c-1f18930e5a58" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>a749c715-8310-4f41-9895-201762e18af6</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>ke/Xsn9T7QlzXsGVjeXAMQ==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Entities.Host.ServerController.GetServersCallBack(CacheItemArgs cacheItemArgs)
   at DotNetNuke.Common.Utilities.DataCache.GetCachedDataFromRuntimeCache(CacheItemArgs cacheItemArgs, CacheItemExpiredCallback cacheItemExpired)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="1df9336a-f61b-4037-9583-895fbe46df3a" LogFileID="" LogTypeKey="CACHE_REMOVED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>DNN_PublishedTabModules55</PropertyName><PropertyValue>Removed</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="596a05f7-37ee-4a56-a5f9-5e57d90a04ce" LogFileID="" LogTypeKey="CACHE_REMOVED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>DNN_FAVICON0</PropertyName><PropertyValue>Removed</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="8329892b-f837-436a-b3f9-d976a9fa5fcb" LogFileID="" LogTypeKey="SCHEDULER_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>cebe0ae6-b627-4119-ab45-91e6c841ddf8</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>9GcAr6O9VzpgL2rtstwlMA==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at PetaPoco.Database.ExecuteScalar[T](String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteScalar[T](String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Data.DataProvider.AddScheduleHistory(Int32 ScheduleID, DateTime StartDate, String Server)
   at DotNetNuke.Services.Scheduling.Scheduler.CoreScheduler.AddScheduleHistory(ScheduleHistoryItem scheduleHistoryItem)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="432b553e-aeb4-4fc9-b4ae-b1a3a1a03680" LogFileID="" LogTypeKey="CACHE_REMOVED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>DNN_PortalDictionary</PropertyName><PropertyValue>Removed</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="feff0a45-a916-47b1-9f22-a827ea254ad0" LogFileID="" LogTypeKey="SCHEDULER_EVENT_STARTED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>THREAD ID</PropertyName><PropertyValue>9</PropertyValue></LogProperty><LogProperty><PropertyName>TYPE</PropertyName><PropertyValue>DotNetNuke.Services.Search.SearchEngineScheduler, DOTNETNUKE</PropertyValue></LogProperty><LogProperty><PropertyName>SOURCE</PropertyName><PropertyValue>STARTED_FROM_TIMER</PropertyValue></LogProperty><LogProperty><PropertyName>ACTIVE THREADS</PropertyName><PropertyValue>2</PropertyValue></LogProperty><LogProperty><PropertyName>FREE THREADS</PropertyName><PropertyValue>-1</PropertyValue></LogProperty><LogProperty><PropertyName>READER TIMEOUTS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>WRITER TIMEOUTS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>IN PROGRESS</PropertyName><PropertyValue>2</PropertyValue></LogProperty><LogProperty><PropertyName>IN QUEUE</PropertyName><PropertyValue>2</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="b0978b38-353e-4add-b0b5-256c5173af0d" LogFileID="" LogTypeKey="CACHE_REMOVED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>DNN_TabSettings0</PropertyName><PropertyValue>Removed</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="792139f6-8154-40db-8ea4-3885f48d28cc" LogFileID="" LogTypeKey="CACHE_REMOVED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>DNN_DotNetNuke.Modules.DigitalAssets.Components.Controllers.DigitalAssetsController, DotNetNuke.Modules.DigitalAssets</PropertyName><PropertyValue>Removed</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="ac8a4244-3019-45f5-a33c-b673db52b262" LogFileID="" LogTypeKey="SCHEDULER_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>7071f33e-b2b8-4df3-b499-6eb7e59dbf3c</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>MNDVumfS8yMQd/794lY1UQ==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Data.DataProvider.GetScheduleItemSettings(Int32 ScheduleID)
   at DotNetNuke.Services.Scheduling.SchedulingController.GetScheduleItemSettings(Int32 ScheduleID)
   at DotNetNuke.Services.Search.Internals.SearchHelperImpl.GetLastSuccessfulIndexingDateTime(Int32 scheduleId)
   at DotNetNuke.Services.Search.SearchEngineScheduler.DoWork()</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="bb5e02b9-1ea4-4cf1-8d6e-16198cef515c" LogFileID="" LogTypeKey="SCHEDULER_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>fe362f39-1369-4c66-8bb1-630189ad0a38</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>a41FJbvIHpNAUQOF5RBleA==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at PetaPoco.Database.Execute(String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteNonQuery(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Data.DataProvider.UpdateScheduleHistory(Int32 ScheduleHistoryID, DateTime EndDate, Boolean Succeeded, String LogNotes, DateTime NextStart)
   at DotNetNuke.Services.Scheduling.SchedulingController.UpdateScheduleHistory(ScheduleHistoryItem objScheduleHistoryItem)
   at DotNetNuke.Services.Scheduling.Scheduler.CoreScheduler.UpdateScheduleHistory(ScheduleHistoryItem scheduleHistoryItem)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="66d8bf50-ec2b-4d1b-a3ca-c9b255585e6d" LogFileID="" LogTypeKey="SCHEDULER_EVENT_FAILURE" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>THREAD ID</PropertyName><PropertyValue>9</PropertyValue></LogProperty><LogProperty><PropertyName>TYPE</PropertyName><PropertyValue>DotNetNuke.Services.Search.SearchEngineScheduler, DOTNETNUKE</PropertyValue></LogProperty><LogProperty><PropertyName>EXCEPTION</PropertyName><PropertyValue>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</PropertyValue></LogProperty><LogProperty><PropertyName>RESCHEDULED FOR</PropertyName><PropertyValue>8/13/2015 2:36:13 AM</PropertyValue></LogProperty><LogProperty><PropertyName>SOURCE</PropertyName><PropertyValue>STARTED_FROM_TIMER</PropertyValue></LogProperty><LogProperty><PropertyName>ACTIVE THREADS</PropertyName><PropertyValue>1</PropertyValue></LogProperty><LogProperty><PropertyName>FREE THREADS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>READER TIMEOUTS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>WRITER TIMEOUTS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>IN PROGRESS</PropertyName><PropertyValue>1</PropertyValue></LogProperty><LogProperty><PropertyName>IN QUEUE</PropertyName><PropertyValue>3</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="18d12289-f066-4420-8d57-472cd758698e" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of a change to Global.asax.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="50162f01-c92e-442c-8b70-dbd4d26450bb" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>748d0c40-a350-4298-b65b-c53e193e3197</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>Ej2Jqb/E5iIEwen61v7nzg==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at DotNetNuke.Data.PetaPoco.PetaPocoExt.ExecuteReader(Database database, String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteReader(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Data.DataProvider.GetScheduleItemSettings(Int32 ScheduleID)
   at DotNetNuke.Services.Scheduling.SchedulingController.GetScheduleItemSettings(Int32 ScheduleID)
   at DotNetNuke.Services.Search.Internals.SearchHelperImpl.GetLastSuccessfulIndexingDateTime(Int32 scheduleId)
   at DotNetNuke.Services.Search.SearchEngineScheduler.DoWork()</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="cb469b41-0cae-4459-9427-e9e6d48c85be" LogFileID="" LogTypeKey="SCHEDULER_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>50fda1cd-f682-477a-a7af-9cc1f5274781</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>QXq15zTe8JTLR/+6bc7K2w==</ExceptionHash><Message>Collection cannot be null.
Parameter name: c</Message><StackTrace>   at System.Collections.ArrayList..ctor(ICollection c)
   at DotNetNuke.Entities.Portals.PortalController.GetPortals()
   at DotNetNuke.Services.ModuleCache.PurgeModuleCache.DoWork()</StackTrace><InnerMessage /><InnerStackTrace /><Source>mscorlib</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="0e0fe869-0a4b-4c85-8fc6-e4f306c65d3a" LogFileID="" LogTypeKey="SCHEDULER_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>0c811a78-0426-4a08-8fe9-24a7a6f679ab</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>52Xbntw0PPVmsHC/JNNYNQ==</ExceptionHash><Message>A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: SQL Network Interfaces, error: 26 - Error Locating Server/Instance Specified)</Message><StackTrace>   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal&amp; connection)
   at System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   at System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   at System.Data.SqlClient.SqlConnection.Open()
   at PetaPoco.Database.OpenSharedConnection()
   at PetaPoco.Database.Execute(String sql, Object[] args)
   at DotNetNuke.Data.PetaPoco.PetaPocoHelper.ExecuteNonQuery(String connectionString, CommandType type, String sql, Object[] args)
   at DotNetNuke.Data.DataProvider.UpdateScheduleHistory(Int32 ScheduleHistoryID, DateTime EndDate, Boolean Succeeded, String LogNotes, DateTime NextStart)
   at DotNetNuke.Services.Scheduling.SchedulingController.UpdateScheduleHistory(ScheduleHistoryItem objScheduleHistoryItem)
   at DotNetNuke.Services.Scheduling.Scheduler.CoreScheduler.UpdateScheduleHistory(ScheduleHistoryItem scheduleHistoryItem)</StackTrace><InnerMessage /><InnerStackTrace /><Source>.Net SqlClient Data Provider</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="33ef7d1e-e59e-452a-9852-29bd05a2422e" LogFileID="" LogTypeKey="SCHEDULER_EVENT_FAILURE" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>THREAD ID</PropertyName><PropertyValue>16</PropertyValue></LogProperty><LogProperty><PropertyName>TYPE</PropertyName><PropertyValue>DotNetNuke.Services.ModuleCache.PurgeModuleCache, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>EXCEPTION</PropertyName><PropertyValue>Collection cannot be null.
Parameter name: c</PropertyValue></LogProperty><LogProperty><PropertyName>RESCHEDULED FOR</PropertyName><PropertyValue>8/13/2015 2:36:12 AM</PropertyValue></LogProperty><LogProperty><PropertyName>SOURCE</PropertyName><PropertyValue>STARTED_FROM_TIMER</PropertyValue></LogProperty><LogProperty><PropertyName>ACTIVE THREADS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>FREE THREADS</PropertyName><PropertyValue>1</PropertyValue></LogProperty><LogProperty><PropertyName>READER TIMEOUTS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>WRITER TIMEOUTS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>IN PROGRESS</PropertyName><PropertyValue>0</PropertyValue></LogProperty><LogProperty><PropertyName>IN QUEUE</PropertyName><PropertyValue>4</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="300c71c4-dae7-49be-be38-2a5f809e3c16" LogFileID="" LogTypeKey="GENERAL_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:35:47 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>9962fa15-fd44-4bda-be3c-c8f2435a400c</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>QXq15zTe8JTLR/+6bc7K2w==</ExceptionHash><Message>Collection cannot be null.
Parameter name: c</Message><StackTrace>   at System.Collections.ArrayList..ctor(ICollection c)
   at DotNetNuke.Entities.Portals.PortalController.GetPortals()
   at DotNetNuke.Services.ModuleCache.PurgeModuleCache.DoWork()</StackTrace><InnerMessage /><InnerStackTrace /><Source>mscorlib</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="9dea9829-103c-4018-b839-995052b61ac6" LogFileID="" LogTypeKey="SCHEDULER_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:36:00 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue /></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>75a69612-e1f8-4e78-82f9-71c7105c5b02</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion /><PortalId>0</PortalId><UserId>0</UserId><TabId>0</TabId><RawUrl /><Referrer /><UserAgent /><ExceptionHash>F5dqyYioORaO09cSq6SUyQ==</ExceptionHash><Message>Thread was being aborted.</Message><StackTrace>   at System.Threading.Thread.SleepInternal(Int32 millisecondsTimeout)
   at System.Threading.Thread.Sleep(Int32 millisecondsTimeout)
   at System.Threading.Thread.Sleep(TimeSpan timeout)
   at DotNetNuke.Services.Scheduling.Scheduler.CoreScheduler.Start()</StackTrace><InnerMessage /><InnerStackTrace /><Source>mscorlib</Source><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method /></Exception></log>﻿<log LogGUID="3fa1ff2a-1eba-48b0-a414-c4c436420857" LogFileID="" LogTypeKey="SCHEDULER_STOPPED" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="8/13/2015 2:37:01 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="TS-WEB03" LogConfigID=""><LogProperties /></log>﻿<log LogGUID="ce4815cc-471c-40ad-ab35-6ea8633bd0a3" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="10/31/2015 7:12:50 AM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of the hosting environment.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="cd40a0fb-7918-4748-bb77-a70f965afc2c" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="10/31/2015 1:37:57 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of the hosting environment.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="e60d38e5-2c40-485f-bb8d-5a4738a4af1a" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="10/31/2015 4:03:25 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of the hosting environment.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="e302a593-f297-4695-a3c0-a44b662c4f5c" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="11/1/2015 1:20:25 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of the hosting environment.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="12f3f0d5-fd3b-4b43-93f5-ac8619a8af19" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="11/2/2015 9:10:13 AM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of the hosting environment.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="bf3e83f7-28f0-4360-9e61-84e4ad0ac80e" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="11/2/2015 9:43:52 AM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of the hosting environment.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="24f0cbe3-a523-4de9-927f-54458a068532" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="11/2/2015 11:03:36 AM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of the hosting environment.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="38f95492-6085-456c-99ed-ef7be917aa7f" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="11/3/2015 6:10:52 AM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of a change to the application level configuration.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="67899fbb-37a7-43e6-90c7-ed110401da85" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="11/3/2015 6:11:10 AM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="TS-WEB03" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of a change to the application level configuration.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="e8d3de40-45cc-4d4f-a2dc-243ccbc71df3" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="3/28/2016 4:12:35 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="THD-01" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of the hosting environment.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="77501203-61a2-4252-838f-de2c4d4d7af7" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="3/28/2016 7:52:49 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="THD-01" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of the hosting environment.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="4f09e172-e5ed-4666-a6b1-4e26fa7a2cff" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="3/28/2016 8:54:20 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="THD-01" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of the hosting environment.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="9cd56088-b35c-42fb-b3a7-6fcb70fe61b8" LogFileID="" LogTypeKey="APPLICATION_SHUTTING_DOWN" LogUserID="-1" LogEventID="-1" LogUserName="" LogPortalID="-1" LogPortalName="" LogCreateDate="3/28/2016 9:32:39 PM" LogCreateDateNum="0" BypassBuffering="True" LogServerName="THD-01" LogConfigID=""><LogProperties><LogProperty><PropertyName>Shutdown Details</PropertyName><PropertyValue>The AppDomain shut down because of the hosting environment.</PropertyValue></LogProperty></LogProperties></log>﻿<log LogGUID="64402580-86b3-43d1-b1b5-bc543f79ed87" LogFileID="" LogTypeKey="PAGE_LOAD_EXCEPTION" LogUserID="-1" LogEventID="-1" LogUserName="sachindev" LogPortalID="0" LogPortalName="SpaFoo" LogCreateDate="6/6/2016 2:32:51 AM" LogCreateDateNum="0" BypassBuffering="False" LogServerName="THD-01" LogConfigID="6"><LogProperties><LogProperty><PropertyName>AbsoluteURL</PropertyName><PropertyValue>/Default.aspx</PropertyValue></LogProperty><LogProperty><PropertyName>DefaultDataProvider</PropertyName><PropertyValue>DotNetNuke.Data.SqlDataProvider, DotNetNuke</PropertyValue></LogProperty><LogProperty><PropertyName>ExceptionGUID</PropertyName><PropertyValue>3f2a69fe-bf5c-49dd-9788-9af705957f1b</PropertyValue></LogProperty></LogProperties><Exception><AssemblyVersion>7.4.1</AssemblyVersion><PortalId>0</PortalId><UserId>3</UserId><TabId>108</TabId><RawUrl>/My-Profile</RawUrl><Referrer>http://dev.spafoo.com/My-Schedule</Referrer><UserAgent>Mozilla/5.0 (Windows NT 6.1; WOW64; rv:46.0) Gecko/20100101 Firefox/46.0</UserAgent><ExceptionHash>FhVyric/Jx9drWfB5pWQBQ==</ExceptionHash><Message>The stored procedure 'dbo.NS_UserProfile_GetUnseenCount' doesn't exist.</Message><StackTrace /><InnerMessage>The stored procedure 'dbo.NS_UserProfile_GetUnseenCount' doesn't exist.</InnerMessage><InnerStackTrace>   at System.Data.SqlClient.SqlCommand.DeriveParameters()
   at System.Data.SqlClient.SqlCommandBuilder.DeriveParameters(SqlCommand command)
   at Microsoft.ApplicationBlocks.Data.SqlHelperParameterCache.DiscoverSpParameterSet(SqlConnection connection, String spName, Boolean includeReturnValueParameter, Object[] parameterValues)
   at Microsoft.ApplicationBlocks.Data.SqlHelperParameterCache.GetSpParameterSetInternal(SqlConnection connection, String spName, Boolean includeReturnValueParameter)
   at Microsoft.ApplicationBlocks.Data.SqlHelperParameterCache.GetSpParameterSet(String connectionString, String spName, Boolean includeReturnValueParameter)
   at Microsoft.ApplicationBlocks.Data.SqlHelper.ExecuteScalar(String connectionString, String spName, Object[] parameterValues)
   at Netsam.Modules.NS_UserProfile.Data.SqlDataProvider.GetUnseenCount(Int32 UserID)
   at Netsam.Modules.NS_UserProfile.Components.NS_ProfileController.GetUnseenCount(Int32 UserID)
   at Netsam.Modules.NS_UserProfile.UC.ucProvider.get_GetUnseenCount()
   at ASP.desktopmodules_ns_userprofile_uc_ucprovider_ascx.__Render__control1(HtmlTextWriter __w, Control parameterContainer)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.WebControls.WebControl.RenderContents(HtmlTextWriter writer)
   at System.Web.UI.WebControls.WebControl.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at ASP.desktopmodules_ns_userprofile_view_ascx.__Render__control1(HtmlTextWriter __w, Control parameterContainer)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at DotNetNuke.UI.Modules.ModuleHost.RenderContents(HtmlTextWriter writer)
   at System.Web.UI.WebControls.WebControl.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.HtmlControls.HtmlContainerControl.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.HtmlControls.HtmlContainerControl.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.HtmlControls.HtmlContainerControl.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at ASP.portals__default_skins_spafoo_inner_ascx.__Render__control1(HtmlTextWriter __w, Control parameterContainer)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.HtmlControls.HtmlForm.RenderChildren(HtmlTextWriter writer)
   at System.Web.UI.HtmlControls.HtmlContainerControl.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.HtmlControls.HtmlContainerControl.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Control.RenderChildrenInternal(HtmlTextWriter writer, ICollection children)
   at System.Web.UI.Page.Render(HtmlTextWriter writer)
   at System.Web.UI.Control.RenderControlInternal(HtmlTextWriter writer, ControlAdapter adapter)
   at System.Web.UI.Page.ProcessRequestMain(Boolean includeStagesBeforeAsyncPoint, Boolean includeStagesAfterAsyncPoint)</InnerStackTrace><Source /><FileName /><FileLineNumber>0</FileLineNumber><FileColumnNumber>0</FileColumnNumber><Method>System.Data.SqlClient.SqlCommand.DeriveParameters</Method></Exception></log></logs>
